Automattic/vip-go-mu-plugins
# Rename Elasticsearch related files

## Description

Builds on #1480 (which must be reviewed + merged first) to rename the search related files and directories from "elasticsearch" to just "search".

## Checklist

Please make sure the items below have been covered before requesting a review:

- n/a This change has relevant unit tests (if applicable).

## Steps to Test

1. Check out PR.
1. Run some commands, like `wp vip-search health validate-counts`
1. Run some queries - `?s=test`, etc
1. Verify no fatals from bad file paths
